bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/NativeInterface.cs

```diff
@@ -18,6 +18,37 @@ internal enum Status
         /// This code should always be the last code in the collection
         // so that the status codes string can be correctly derived
         ELECTIONGUARD_STATUS_UNKNOWN
+
+
+    }
+
+    static internal class StatusExtensions
```

Comment:
i left this here right next to the enum because it needs to be updated if the enumeration changes at all and i figured it would be easy to miss if it was in its own file.